styles.css

```diff
@@ -10,10 +19,19 @@
     color:var(--secondary-color)
 }
 
+.price{
+    border:1px solid black
+}
+
 .footer {
     background-color: var(--primary-color);
     padding: 3rem 1rem;
     text-align: center;
     color: white;
     border-top-right-radius: 1rem;
+}
+
+.zero-bottom{
+    position: fixed;
+    bottom: 0;
```

Comment:
![image](https://user-images.githubusercontent.com/67817330/154215341-45625801-3c3a-4cb4-a53c-a357df46e4e3.png)
Hey Shlok, the footer is blogging my view its rather too thick for the footer , maybe tone down the padding to couple of pixels , also its not advisable to have a fixed footer.

Reply:
Hey,

Thanks for the advice, I have removed the fixed positioning and used absolute instead.